fem/fespace.cpp

```diff
@@ -2803,9 +2803,17 @@ FiniteElementSpace::GetElementDofs(int elem, Array<int> &dofs) const
 
 const FiniteElement *FiniteElementSpace::GetFE(int i) const
 {
-   if (i < 0 || !mesh->GetNE()) { return NULL; }
-   MFEM_VERIFY(i < mesh->GetNE(),
-               "Invalid element id " << i << ", maximum allowed " << mesh->GetNE()-1);
+   if (i < 0 || i >= mesh->GetNE())
+   {
+      if (mesh->GetNE() == 0)
+      {
+         MFEM_ABORT("Empty MPI partition!!! NOTE: FESpace::GetFE has been "
+                    "updated to abort instead of returning NULL. The user must "
```

Comment:
Just a minor suggestion, but I suppose after this is merged, the comment about the being updated may be confusing, if the reader does not know the history of this function. It seems simpler and more straightforward to say only that empty partitions are not permitted within this function. That makes the comment about the current state of the code, not the history.